gtsam/geometry/Pose3.h

```diff
@@ -379,6 +379,14 @@ class GTSAM_EXPORT Pose3: public LieGroup<Pose3, 6> {
     return std::make_pair(0, 2);
   }
 
+    /**
+   * @brief Spherical Linear interpolation between *this and other
+   * @param s a value between 0 and 1.5
+   * @param other final point of interpolation geodesic on manifold
+   */
+  Pose3 interp(double t, const Pose3& other, OptionalJacobian<6, 6> Hx = boost::none,
```

Comment:
On second thought, I actually prefer this since `Lie::interpolate` is a free function. Can you please rename this to `slerp` similar to `Rot3`?

Reply:
Hmm, isn't "slerp" referring to interpolation of rotations on the quaternion unit sphere specifically?